wdl/tasks/VariantCalling/TRGT.wdl

```diff
@@ -0,0 +1,90 @@
+version 1.0
+
+workflow runTRGT {
+
+  meta {
+    description: "Uses TRGT to size TRs in a bam file."
+  }
+
+  input {
+    File input_bam
+    File input_bam_bai
+    String basename = basename(input_bam, ".bam")
+    String output_gs_path
+    File ref_dict
+    File ref_fasta
+    File ref_fasta_index
+    File repeatCatalog = "https://zuchnerlab.s3.amazonaws.com/RepeatExpansions/TRGT/adotto_TRregions_TRGTFormatWithFlankingSeq_v1.0.bed"
```

Comment:
I've never tried running workflows that try to localize HTTP files. So I worry this default value will fail the workflow's executions.

This points to a larger issue: where should these annotations be stored?

I don't mean we should resolve this here, but just getting it out there for us to think about it.

Reply:
You are correct. Apparently you can import WDLs from https, but can't localize files that way. But yes, this is obviously just because I didn't have a good place to store the annotation file. It is about 275MB, so not really appropriate for Github. Ideally, we would put it somewhere on gs. But I don't have an account where I can make a file publicly available there. 

For now, I went ahead and moved this to a wget command within the Dockerfile so the annotation is built in to the image (and then referenced locally). But if we can make a place for me to host the annotation on gs, that would obviously be a much better solution as the annotation will certainly change over time and re-building the docker image for that is clearly not optimal.